apps/site/.markdownlint.json

```diff
@@ -0,0 +1,19 @@
+{
+  "default": true,
+  "MD001": true,
+  "MD009": false,
```

Comment:
Why do we want to disable this one? Is there a use case where this is necessary?

`"MD009": false - Disables the rule preventing trailing spaces`

Reply:
Yes, exactly. There are around 100 pages that have some space at the end of the line and it returns an error.